christfellowshipchurch/web-app-v2
# Groups resources tags

### About
This PR makes the process of loading the group resources when adding them to a group faster by query them as TaggedItems when a category is selected.

_TODO: Tag all resources so they show up before merging into main_
### Test Instructions
1. Open the testing link and log in.
2. Click on your profile picture on the top right and go to your groups.
3. Open any of your groups, click on manage group and try to add a resource.
4. Compare the difference of loading speed compared to live website.
5. Also check that it allows to add and remove resources, and that if a resource is already added to the group resources that it does not show as available.

### Screenshots
<img width="1440" alt="Screenshot 2023-09-22 at 2 05 13 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/febd8fa1-912d-420a-94b4-1d119f982ba9">


### Closes Tickets
[CFDP-2612](https://christfellowshipchurch.atlassian.net/browse/CFDP-2612)


[CFDP-2612]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2612?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
